web_app/db/seed_data.py

```diff
@@ -0,0 +1,100 @@
+import os 
+import sys
+from sqlalchemy import create_engine  
+from sqlalchemy.orm import sessionmaker  
+from faker import Faker  
+# from web_app.db.models import User, Position, AirDrop, TelegramUser
+# from web_app.db.database import Base
+from .models import User, Position, AirDrop, TelegramUser  
+from .database import Base
+from dotenv import load_dotenv
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(__file__)))) 
+load_dotenv()
+
+# Database connection settings  
+DB_USER = os.getenv("DB_USER")
+DB_PASSWORD = os.getenv("DB_PASSWORD")
+DB_NAME = os.getenv("DB_NAME")
+DB_HOST = os.getenv("DB_HOST")
+DB_PORT = os.getenv("DB_PORT")
+
+DATABASE_URL = "postgresql+psycopg2://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
+
+# Create a new database session  
+engine = create_engine(DATABASE_URL)  
+Session = sessionmaker(bind=engine)  
+session = Session()  
```

Comment:
REmove this code:
```
# Database connection settings  
DB_USER = os.getenv("DB_USER")
DB_PASSWORD = os.getenv("DB_PASSWORD")
DB_NAME = os.getenv("DB_NAME")
DB_HOST = os.getenv("DB_HOST")
DB_PORT = os.getenv("DB_PORT")

DATABASE_URL = "postgresql+psycopg2://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"

# Create a new database session  
engine = create_engine(DATABASE_URL)  
Session = sessionmaker(bind=engine)  
session = Session() 
```
import it from [here](https://github.com/djeck1432/spotnet/blob/main/web_app/db/database.py#L30)